enterprise/cmd/frontend/internal/codeintel/resolvers/graphql/resolver.go

```
@@ -294,6 +294,22 @@ func makeGetUploadsOptions(ctx context.Context, args *gql.LSIFRepositoryUploadsQ
 		return store.GetUploadsOptions{}, err
 	}
 
+	var dependencyOf int64
+	if args.DependencyOf != nil {
+		dependencyOf, err = unmarshalLSIFUploadGQLID(*args.DependencyOf)
```

Technically, this doesn't guard against use of a wrong kind in the ID, I think. But not a problem I guess.

---

Should it throw when the dependency doesn't exist? Currently silently returns empty connections, afaict

---

Ah maybe we should check `dependencyOf != 0` when `args.DependencyOf != ""`. Empty connections are probably alright since when we make this API request today we do it in the context of an LSIF upload page.

Will follow-up both of these issues post-merge.